stubs/annoy/annoy/__init__.pyi

```diff
@@ -0,0 +1,44 @@
+from typing import Sequence, Tuple, overload
+from typing_extensions import Literal
+
+_Vector_Float = Sequence[float]
+_Vector_Int = Sequence[int]
+
+class AnnoyIndex:
+    f: int
+    def __init__(self, f: int, metric: Literal["angular", "euclidean", "manhattan", "hamming", "dot"]) -> None: ...
+    def load(self, fn: str, prefault: bool = ...) -> Literal[True]: ...
+    def save(self, fn: str, prefault: bool = ...) -> Literal[True]: ...
+    @overload
+    def get_nns_by_item(self, i: int, n: int, search_k: int = ..., include_distances: Literal[False] = ...) -> _Vector_Int: ...
+    @overload
+    def get_nns_by_item(
+        self, i: int, n: int, search_k: int, include_distances: Literal[True]
+    ) -> Tuple[_Vector_Int, _Vector_Float]: ...
```

Comment:
The return types are lists, not arbitrary sequences. We prefer to be more precise in return types, so that people who use the return value as a list don't get spurious errors.

Reply:
Fixed, thanks for the feedback